middlewares/auth.js

```diff
@@ -1,14 +1,15 @@
 // const jwt = require('jsonwebtoken');
 const passport = require('passport')
-const {Strategy, ExtractJwt} = require('passport-jwt')
+const passportJWT = require('passport-jwt')
 const User = require('../models/schemas/user')
 require('dotenv').config()
 const secret = process.env.SECRET;
-
+const ExtractJWT = passportJWT.ExtractJwt;
```

Comment:
Umieszczaj importy z bibliotek zewnętrznych na początku, a następnie lokalne importy. Oddzielaj je pustymi liniami.
Alfabetyczne Sortowanie: Sortuj importy alfabetycznie w obrębie każdej grupy. Ułatwia to znalezienie potrzebnych importów.
Unikaj Nadmiernego Dzielania: Staraj się nie importować więcej niż potrzebujesz. Zamiast import *, lepiej jest importować tylko konkretne funkcje czy komponenty.